contracts/lease/src/position/mod.rs

```diff
@@ -238,78 +239,50 @@ mod test_check {
     type TestLpn = Usdc;
 
     const LEASE_AMOUNT: Coin<TestCurrency> = Coin::new(1000);
-    const LEASE_AMOUNT_LPN: Coin<TestLpn> = Coin::new(1000);
-    const MIN_DUE_AMOUNT: Coin<TestLpn> = Coin::new(100);
-    const MIN_DUE_AMOUNT_ASSET: Coin<TestCurrency> = Coin::new(100);
-    const MIN_ASSET: Coin<TestLpn> = Coin::new(100);
-    const MIN_SELL_ASSET: Coin<TestLpn> = Coin::new(1);
     const PRICE_TEST_LPN: Coin<TestLpn> = Coin::new(1_000);
     const PRICE_TEST_CURRENCY: Coin<TestCurrency> = Coin::new(1_000);
 
     #[test]
     fn no_debt() {
         let warn_ltv = Percent::from_permille(11);
-        let spec = liability_with_first(warn_ltv, 100.into(), MIN_ASSET, MIN_SELL_ASSET);
+        let spec = liability_with_first(warn_ltv, 100.into(), 1.into(), 1.into());
```

Comment:
please keep the fn names changed as their purpose does. `liability_with_*` no more creates Liability instances

Reply:
do not remember that one should read the code as a chunk of prose